server/__init__.py

```diff
@@ -122,7 +122,7 @@ def create_app():
     app.config['BABEL_TRANSLATION_DIRECTORIES'] = 'i18n'
 
     if not cfg.TEST:
-        timeout = 120  # seconds
+        timeout = 60  # seconds
```

Comment:
i don't remember why we bumped up the timeout - were there long API calls that we rely on? even so, best to do this in a separate PR

Reply:
Done